CSVSource/src/main/docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM openjdk:11-jre-slim
```

Comment:
Same change as done for camel connector.  This change is for all the java-based connectors